extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -249,25 +249,40 @@ class OHIFCornerstoneViewport extends Component {
   render() {
     let childrenWithProps = null;
 
+    if (!this.state.viewportData) {
+      return <></>;
+    }
+    const { viewportIndex } = this.props;
+    const {
+      imageIds,
+      currentImageIdIndex,
+      frameRate = 0, // Sometimes available here
```

Comment:
`frameRate` appears unused?

Reply:
Not sure if this derived from metaData and should override the default/initial value. It can be hard to trace where data is coming from, so I left it for now. I can remove it

Reply:
It would come from the FrameTime in the instance, for example.